### PR TITLE
[ADD] module sale_owner_stock_sourcing

### PR DESCRIPTION
**Pull request by yvaucher**
This modules allows to choose stock ownership on sale order line

Dependencies:
- [x] OCA/stock-logistics-workflow#44

### CODE REVIEW

**Comment by lepistone** [comment]
Note: the red build is due to OCA/stock-logistics-workflow#44 not being merged yet. Tests pass with it.

File: sale_owner_stock_sourcing/README.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Sale Owner Stock Sourcing
+=========================
+
+This module allow you to choose for every sale order line the owner of the
```
Comment (gurneyalex) [inline sale_owner_stock_sourcing/README.rst]: s/allow/allows/

File: sale_owner_stock_sourcing/README.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+stock that will be dispatched.  By choosing an owner, the generated deliveries
+will then look for stock of the specified partner.  If no one is available,
```
Comment (gurneyalex) [inline sale_owner_stock_sourcing/README.rst]: s/of the/belonging to/
s/no one/none/

File: sale_owner_stock_sourcing/README.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+then the picking will be in waiting availability until stocks of the proper
```
Comment (gurneyalex) [inline sale_owner_stock_sourcing/README.rst]: s/in//
s/of the/with/

File: sale_owner_stock_sourcing/README.rst [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+ownership are replenished.
+
+Specifying an owner on a line, forces to use the stock of this owner.  Letting
```
Comment (gurneyalex) [inline sale_owner_stock_sourcing/README.rst]: s/,//
s/Letting/Leaving/

File: sale_owner_stock_sourcing/README.rst [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+the owner empty allows to use stock (quant) without owner.
+
+Note: pickings and moves both have a owner field. Here we only propagate the
```
Comment (gurneyalex) [inline sale_owner_stock_sourcing/README.rst]: s/a owner/an owner/

File: sale_owner_stock_sourcing/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+ "version": "0.1",
+ "author": "Camptocamp",
+ "license": "AGPL-3",
+ "category": "Purchase Management",
+ 'complexity': "normal",
+ "images": [],
+ "website": "http://www.camptocamp.com",
+ "depends": ['sale_stock',
+             'stock_ownership_availability_rules',
+             #  'purchase_vendor_consignment_stock',
```
Comment (gurneyalex) [inline sale_owner_stock_sourcing/__openerp__.py]: remove unneeded dependency

File: sale_owner_stock_sourcing/__openerp__.py [hunk 7]
```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+#
+#
+#    Author: Yannick Vaucher, Leonardo Pistone
+#    Copyright 2014 Camptocamp SA
```
Comment (gurneyalex) [inline sale_owner_stock_sourcing/__openerp__.py]: s/2014/2014-2015/

File: sale_owner_stock_sourcing/model/procurement.py [hunk 8]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+#    Author: Leonardo Pistone
+#    Copyright 2014 Camptocamp SA
```
Comment (gurneyalex) [inline sale_owner_stock_sourcing/model/procurement.py]: update year

File: .travis.yml [hunk 9]
```diff
@@ -19,6 +19,7 @@ install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
+  - git clone https://github.com/OCA/stock-logistics-workflow -b ${VERSION} $HOME/stock-logistics-workflow
```
Comment (gurneyalex) [inline .travis.yml:22]: could you add the following line to a new text file at the root of the repository called oca-dependencies.txt:

```
stock-logistics-workflow
```

**Comment by gurneyalex** [comment]
Reviewed:
- a few spelling / grammar improvements for the description of the module. 
- maybe you can update the (c) years
- I think installing this module should enable the group allowing to edit the sale order lines in form view (otherwise you don't get access to the new field). 

Great work!

**Comment by gurneyalex** [comment]
And maybe add a .pot file in i18n/

**Comment by lepistone** [comment]
@gurneyalex I addressed your remarks.

**Comment by lepistone** [comment]
Of course I :+1: @yvaucher's initial work.

**Pull request merged**